src/protocol/protocol.yml

```diff
@@ -220,6 +220,52 @@ InterceptedResponse:
           value: string
 
 
+FetchRequest:
+  type: interface
+
+  initializer:
+    url: string
+    method: string
+    postData: binary?
+    headers:
+      type: array
+      items:
+        type: object
+        properties:
+          name: string
+          value: string
+    redirectedFrom: FetchRequest?
```

Comment:
I don't see this being used and I don't really want to see it to keep things simple... Should we just follow redirects on the driver side w/o client knowing?